clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoClient.java

```diff
@@ -537,7 +555,7 @@ public GravitinoClient build() {
           metalakeName != null && !metalakeName.isEmpty(),
           "The argument 'metalakeName' must be a valid name");
```

Comment:
I think the checks on metalakeName will stop the new constructor from being called if metalake name is null or empty.